packages/storybook/src/utilities/storybook.ts

```diff
@@ -181,6 +181,9 @@ export const iconDescription = 'Set `slot="start"` to include an icon before the
 export const disabledDescription = (options: {
     componentName: string
 }): string => `Styles the ${options.componentName} as disabled and prevents focus and user interaction.`;
+export const readonlyDescription = (options: {
+    componentName: string
+}): string => `Disallows input on the ${options.componentName} while maintaining enabled appearance.`;
```

Comment:
We say `while maintaining enabled appearance` but that doesn't seem accurate, since we change the border/underline color (same as for disabled styling). Minor, but figured I'd comment to see if others think we should remove that wording or not.

Reply:
I think I might have written this originally in the hopes of distinguishing it from `disabled`, but I agree it's not really accurate. How about this, borrowing from `disabled` above?
```suggestion
}): string => `Styles the ${options.componentName} as readonly and prevents the user from changing the value.`;
```

Alternatively we could just copy the MDN definitions, but they're a bit wordier than we usually go with:
>The Boolean readonly attribute, when present, makes the element not mutable, meaning the user can not edit the control.
>The Boolean disabled attribute, when present, makes the element not mutable, focusable, or even submitted with the form. The user can neither edit nor focus on the control, nor its form control descendants.